assets/svg/icon-user.svg

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Looking at this SVG closer, it seems the only thing it gives us is a large FontAwesome icon... is there a reason we need this as SVG image? If so, we probably want to vectorise the glyph rather than relying on the FontAwesome font. If not, we already have FontAwesome loaded so we could just show this in HTML with a `<span>` without any need for an SVG file